terraform/modules/stack/base/outputs.tf

```diff
@@ -65,11 +65,20 @@ output "rds_postgres_security_group" {
 
 /* RDS Bosh Instance */
 output "bosh_rds_url" {
```

Comment:
Failing to see why given it's just the outputs? Why would we need to touch state?